src/Admin/Content/AdminPayload.php

```diff
@@ -81,5 +82,18 @@ public function __invoke(Document $document, Request $request)
 
         $document->payload['phpVersion'] = PHP_VERSION;
         $document->payload['mysqlVersion'] = $this->db->selectOne('select version() as version')->version;
+
+        /**
+         * Used in the admin user list. Implemented as this as it matches the API in flarum/statistics.
+         * If flarum/statistics ext is enabled, it will override this data with its own stats.
+         *
+         * This allows the front-end code to be simpler and use one single source of truth to pull the
+         * total user count from.
+         */
+        $document->payload['statistics'] = [
```

Comment:
modelStatistics might be better. This is private API so we can always change it later.

Reply:
I chose this because it matches with what's currently used in `flarum/statistics`. Should we change it in that extension too?